### DISCUSSION TITLE
增强功能

### DISCUSSION THREAD

**Discussion by willrealize** [HEIGE-PCloud/DoIt#384] [Ideas]
在导航栏中的 **文档** 选项, 其实是从 **分类** 中单独拿出来的, 它依然会显示在 **分类** 中, 我想能不能像 **系列** 那样, 把它作为一个单独的功能, 另外, 如果做成文档功能, 可以考虑把文章页面的左侧空白部分, 做成文章名目录, 就像右边的目录, 这样看起来更像是文档😊

**Comment by ghost** [comment]
是否可以参考 gitbook、语雀 的样式？

**Reply by willrealize** [reply]
都可以的😊

**Comment by HEIGE-PCloud** [comment]
DoIt 是为个人博客所设计的主题。如果需要编写与展示文档的话，也许选择一些专门为文档所设计的主题会更加合适。

https://themes.gohugo.io/tags/docs/

**Reply by willrealize** [reply]
比起其他的主题, 我还是更喜欢这个😊
